rfcs/css_build_system.md

```diff
@@ -0,0 +1,15 @@
+# RFC 136: Remove the CSS build system
+
+## Summary
+
+Remove css/tools/ and the special lint rules for css/.
+
+## Details
+
+http://test.csswg.org/suites/ hasn't been updated since January.
```

Comment:
https://github.com/w3c/csswg-drafts/issues/6896 makes it sound like there have been serious problems there for much longer. (And indeed I get the error given in https://github.com/w3c/csswg-drafts/issues/6896#issuecomment-996047519 when I try to build it locally.)